.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,53 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG] "
+labels: 'bug'
+assignees: 'black-bunny-brigade'
+
+---
+
+**IMPORTANT: please make sure you ask yourself all intro questions and fill all sections of the template.**
+
+**Before we start...:**
+
+- [ ] I checked the documentation and found no answer
+- [ ] I checked to make sure that this issue has not already been filed
+- [ ] I'm reporting the issue to the correct repository (for multi-repository projects)
+
+
+**Version, Branch, or Commit:**
+
+Inform what version, branch, commit of Skunk you are using.
+
+**Expected behavior:**
+
+Please include a detailed description of the behavior you were expecting when you encountered this issue.
+
+**Actual behavior:**
+
+Please include a detailed description of the actual behavior of the application.
+
+**Steps to reproduce:**
+
+How do I achieve this behavior? Use the following format to provide a step-by-step guide:
+
+1. Step 1: ...
+2. Step 2: ...
+
+**Context and environment:**
+
+Provide any relevant information about your setup (Customize the list accordingly based on what info is relevant to this project)
+
+1. Version of the software the issue is being opened for.
+2. Operating System
+3. Operating System version
+4. Ruby version
+
+_Delete any information that is not relevant._
+
+**Logs**
+
+Include relevant log snippets or files here.
+
+**I will abide by the [code of conduct] (CODE_OF_CONDUCT.md)**
```

Comment:
this is linking to a file that we don't have, should we add it too in this PR?

Reply:
This should be fixed after https://github.com/fastruby/rails-template/pull/11 is merged